Description: specify Debian path and group names in password plugin
Forwarded: not-needed
Author: Jérémy Bobbio <lunar@debian.org>

diff --git a/plugins/password/README b/plugins/password/README
index 81e4f1e..bdb8264 100644
--- a/plugins/password/README
+++ b/plugins/password/README
@@ -133,7 +133,8 @@
 
  Installation:
 
- Change into the drivers directory. Edit the chgsaslpasswd.c file as is
+ Change into the drivers directory. Copy and edit
+ /usr/share/roundcube-plugins/examples/chgsaslpasswd.c as is
  documented within it.
 
  Compile the wrapper program:
@@ -200,7 +201,8 @@
  ----------------------------
 
  As in sasl driver this one allows to change password using shell
- utility called "virtualmin". See drivers/chgvirtualminpasswd.c for
+ utility called "virtualmin". See
+ /usr/share/doc/roundcube-plugins/examples/chgvirtualminpasswd.c for
  installation instructions. See also config.inc.php.dist file.
 
 
@@ -225,8 +227,9 @@
  Driver that adds functionality to change the systems user password via 
  the 'chpasswd' command. See config.inc.php.dist file.
 
- Attached wrapper script (chpass-wrapper.py) restricts password changes
- to uids >= 1000 and can deny requests based on a blacklist.
+ Attached wrapper script
+ (/usr/share/doc/roundcube-plugins/examples/chpass-wrapper.py) restricts
+ password changes to uids >= 1000 and can deny requests based on a blacklist.
 
 
  2.12.  LDAP - no PEAR (ldap_simple)
diff --git a/plugins/password/drivers/chgsaslpasswd.c b/plugins/password/drivers/chgsaslpasswd.c
index bcdcb2e..2c3bbf9 100644
--- a/plugins/password/drivers/chgsaslpasswd.c
+++ b/plugins/password/drivers/chgsaslpasswd.c
@@ -8,7 +8,7 @@
 
 /* INSTALLING:
   gcc -o chgsaslpasswd chgsaslpasswd.c
-  chown cyrus.apache chgsaslpasswd
+  chown cyrus:www-data chgsaslpasswd
   strip chgsaslpasswd
   chmod 4550 chgsaslpasswd
 */
diff --git a/plugins/password/drivers/chgvirtualminpasswd.c b/plugins/password/drivers/chgvirtualminpasswd.c
index 4e2299c..12989c6 100644
--- a/plugins/password/drivers/chgvirtualminpasswd.c
+++ b/plugins/password/drivers/chgvirtualminpasswd.c
@@ -7,7 +7,7 @@
 
 /* INSTALLING:
   gcc -o chgvirtualminpasswd chgvirtualminpasswd.c
-  chown root.apache chgvirtualminpasswd
+  chown root:www-data chgvirtualminpasswd
   strip chgvirtualminpasswd
   chmod 4550 chgvirtualminpasswd
 */
